.eslintrc.js

```diff
@@ -1,15 +1,12 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 2020
+    ecmaVersion: 'latest'
```

Comment:
We target the latest two browser versions, so I think `latest` makes sense. We certainly aren't targeting 2020.